course_discovery/apps/course_metadata/search_indexes.py

```diff
@@ -260,7 +260,7 @@ def read_queryset(self, using=None):
         )
 
     def index_queryset(self, using=None):
-        return filter_visible_runs(super().index_queryset(using=using))
+        return super().index_queryset(using=using)
```

Comment:
Have we tested this core change?

Reply:
yes

Reply:
![Uploading Screen Shot 2021-05-21 at 11.30.51 AM.png…]()